### PR TITLE
Fix style issue and enable scalastyle check

### PR DESCRIPTION
**Pull request by janisz**

### CODE REVIEW

File: src/main/scala/mesosphere/marathon/upgrade/DeploymentPlanReverter.scala [hunk 1]
```diff
@@ -55,85 +55,44 @@ private[upgrade] object DeploymentPlanReverter {
...
-          result.update(
-            newGroup.id,
-            group => group.copy(dependencies = group.dependencies -- newGroup.dependencies),
-            version)
-        case _ =>
-          // still contains apps/groups, so we keep it
-          result
-      }
-    }
+sealed trait DeploymentPlanGroupChangesReverter {
```
Comment (aquamatthias) [inline src/main/scala/mesosphere/marathon/upgrade/DeploymentPlanReverter.scala]: What is the reasoning behind this trait? Could we define the methods directly in the object?

**Comment by aquamatthias** [comment]
@janisz Sorry for getting back so late - please rebase. One question if we can define all methods in the object directly? Otherwise looks good.

File: src/main/scala/mesosphere/marathon/upgrade/DeploymentPlanReverter.scala (hunk 1 above)
Comment (janisz) [inline src/main/scala/mesosphere/marathon/upgrade/DeploymentPlanReverter.scala]: I was thinking about separating group from app revert but yes, this trait is redundant.

**Comment by ssk2** [comment]
ok to tests

**Comment by aquamatthias** [comment]
@janisz Our test infrastructure is not stable, so I was not able to merge. Can you rebase again please?

**Comment by jeschkies** [comment]
@janisz could you rebase once more?

**Comment by jeschkies** [comment]
It seems stylecheck has been remove with 31426bd1f7b091b78748ea7fd83c497db3dfea9e

**Pull request closed without merging**